### ISSUE TITLE
Design customization

### ISSUE THREAD

**Issue by SIMC4774** [COVID-science-engine/crowdsourced-papers-prototype#4]
- Design System created in Figma, to support scalable development of the platform. (Easy theme changes, and component changes/additions).

- Introduced Blue as the primary color theme (Tailwind standards)

- Paragraph color changed to gray-600 (tailwind default color) - darkened to comply with WCAG accessibility standards, improving readability.

- Heading hierarchy introduced in styling, to support scaling of the platform. Adopted the current H1 from helpwithcovid, for later introduction of a landing page. **Needs to be aligned with frontend.**

- UX change in prototype: h4 (Molecular targets, Key Findings and Context), should always be followed by a line break before paragraph text, to support scannability, and create logical hierarchy. 

See Design System and wireframes at the Figma link here: https://www.figma.com/file/RtHSp2qSI9TJfzxjRmLCOj/Design-System?node-id=0%3A1

**Comment by MatMoore** [comment]
I've changed the colours and made that UX change: https://covid-science-engine-prototype.herokuapp.com/

Do you have any thoughts on how to go about aligning the heading hierarchy?

I was wondering if it would help to start using PostCSS to organise the CSS as described [here](https://tailwindcss.com/docs/using-with-preprocessors#build-time-imports) rather than combining lots of tailwind classes in the HTML.

**Comment by SIMC4774** [comment]
Yes, I think if we can have a talk about it tomorrow (i am not available before late tonight). 

It could be an idea, but when i read the tailwind instruction I pay attention to this phrase: _"It's important to note that you don't need to use a preprocessor with Tailwind — you typically write very little CSS on a Tailwind project anyways so using a preprocessor just isn't as beneficial as it would be in a project where you write a lot of custom CSS."_

I think that is true for our project, but if you see benefits, it is your call. It is no problem for me to dive into PostCSS. What do you think?

**Comment by MatMoore** [comment]
Sure, I should be available tomorrow morning so I'll join the discord when I'm free.

I've never used PostCSS before but I'm happy to spend a bit of time experimenting with it and if it's more complicated than the current approach then we can keep it as is. 

I'm hoping it would let us write something like

```
.h1 {
  @apply text-2xl sm:text-4xl font-bold tracking-tight leading-tight;
}
```

and then we wouldn't have to repeat that in every template.

**Comment by MatMoore** [comment]
@SIMC4774 do you think this issue can be closed now? It looks like this will be an ongoing thing, and one of the next things for me to pick up will be implementing the landing page.

Regarding the heading hierarchy, I've summarised the outstanding issues in this comment: https://github.com/COVID-science-engine/crowdsourced-papers-prototype/pull/12#issuecomment-612889800
